### PR TITLE
Add compat data for Metadata

### PR DESCRIPTION
**Pull request by bunnybooboo**
https://developer.mozilla.org/en-US/docs/Web/API/Metadata

### CODE REVIEW

File: api/Metadata.json [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+  "api": {
+    "Metadata": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/Metadata",
+        "support": {
+          "webview_android": {
+            "version_added": false
+          },
+          "chrome": {
+            "version_added": "13"
```
Comment (maboa) [inline api/Metadata.json]: I think you need to add `"prefix": "webkit"` here so maybe something like:

```
"prefix": "webkit",
"version_added": "13"
```

File: api/Metadata.json [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+          },
+          "chrome_android": {
+            "version_added": "0.16"
```
Comment (maboa) [inline api/Metadata.json]: Again, this is prefixed and that should probably be represented.

File: api/Metadata.json [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+      },
+      "modificationTime": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Metadata/modificationTime",
+          "support": {
+            "webview_android": {
+              "version_added": false
+            },
+            "chrome": {
+              "version_added": "13"
```
Comment (maboa) [inline api/Metadata.json]: `"prefix": "webkit"` ?

File: api/Metadata.json [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+            },
+            "chrome_android": {
+              "version_added": "0.16"
```
Comment (maboa) [inline api/Metadata.json]: `"prefix": "webkit"` ?

File: api/Metadata.json [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+      },
+      "size": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Metadata/size",
+          "support": {
+            "webview_android": {
+              "version_added": false
+            },
+            "chrome": {
+              "version_added": "13"
```
Comment (maboa) [inline api/Metadata.json]: `"prefix": "webkit"` ?

File: api/Metadata.json [hunk 6]
```diff
@@ -0,0 +1,157 @@
...
+            },
+            "chrome_android": {
+              "version_added": "0.16"
```
Comment (maboa) [inline api/Metadata.json]: `"prefix": "webkit"` ?

File: api/Metadata.json [hunk 7]
```diff
@@ -0,0 +1,163 @@
...
+        "support": {
+          "webview_android": {
+            "version_added": false
+          },
+          "chrome": {
+            "version_added": "13",
+            "prefix": "webkit"
+          },
+          "chrome_android": {
+            "version_added": "0.16",
```
Comment (ddbeck) [inline api/Metadata.json]: I think this version number is unlikely. I'd rather see this as `true`

File: api/Metadata.json [hunk 8]
```diff
@@ -0,0 +1,163 @@
...
+      "modificationTime": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Metadata/modificationTime",
+          "support": {
+            "webview_android": {
+              "version_added": false
+            },
+            "chrome": {
+              "version_added": "13",
+              "prefix": "webkit"
```
Comment (ddbeck) [inline api/Metadata.json]: I don't think this is doubly prefixed (e.g., `webkitMetadata.webkitmodificationTime`). I think this prefix can be removed

File: api/Metadata.json [hunk 9]
```diff
@@ -0,0 +1,163 @@
...
+            },
+            "chrome_android": {
+              "version_added": "0.16",
```
Comment (ddbeck) [inline api/Metadata.json]: I think this should be `true` as well

File: api/Metadata.json [hunk 10]
```diff
@@ -0,0 +1,163 @@
...
+              "prefix": "webkit"
```
Comment (ddbeck) [inline api/Metadata.json]: This could be removed too

File: api/Metadata.json [hunk 11]
```diff
@@ -0,0 +1,163 @@
...
+      "size": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Metadata/size",
+          "support": {
+            "webview_android": {
+              "version_added": false
+            },
+            "chrome": {
+              "version_added": "13",
+              "prefix": "webkit"
```
Comment (ddbeck) [inline api/Metadata.json]: This could be removed too

File: api/Metadata.json [hunk 12]
```diff
@@ -0,0 +1,163 @@
...
+            },
+            "chrome_android": {
+              "version_added": "0.16",
```
Comment (ddbeck) [inline api/Metadata.json]: I think this should be `true` as well

File: api/Metadata.json [hunk 13]
```diff
@@ -0,0 +1,163 @@
...
+              "prefix": "webkit"
```
Comment (ddbeck) [inline api/Metadata.json]: This could be removed too

**Pull request merged**